nevis/_util.py

```diff
@@ -110,3 +112,141 @@ def print_result(x, y, z):
     photo = hill.photo()
     if photo:
         print('  ' + photo)
+
+
+def generate_kml(file, labels=None, trajectory=None, points=None,):
+    """
+    Generate a KML (Keyhole Markup Language, used to display geographic data
+    in an Earth browser such as Google Earth) file from given data.
+
+    Arguments:
+
+    ``file``
+        The name of the file to write to.
+    ``labels``
+        An optional dictionary mapping string labels to points (tuples in
+        meters or Coords objects) that will be marked on the map.
+    ``trajectory``
+        An optional array of shape ``(n_points, 2)`` indicating the trajectory
+        followed to get to Ben Nevis (points specified in meters).
+    ``points``
+        An optional array of shape ``(n_points, 2)`` indicating points on the
+        map (points specified in meters).
+    """
+
+    def make_coords(x, y):
```

Comment:
Should we add this to `Coords`? e.g. as a property `longlat_string` or `longlat_str` ?

Reply:
I guess this is not necessary because this string format is quite ad hoc for KML files, and introducing it would kind of break our convention of using `lat, lon` (instead of `lon, lat`) in the module.

Reply:
ok